db/wide/wide_columns_helper.h

```diff
@@ -11,11 +11,23 @@
 #include "rocksdb/wide_columns.h"
 
 namespace ROCKSDB_NAMESPACE {
+
 class WideColumnsHelper {
  public:
-  static void DumpWideColumns(const WideColumns& columns, std::ostream& oss,
```

Comment:
If you rebase to the latest, you should get the fix. I addressed your comment in #11777

Reply:
Seems like it's still `oss` in the header file on mainline, see https://github.com/facebook/rocksdb/blob/main/db/wide/wide_columns_helper.h#L16

Reply:
Ah, you are right. looks like I only fixed in `.cc`. Thanks for catching this.